### ISSUE TITLE
Image not Loading for Hikvision IP Camera

### ISSUE THREAD

**Issue by cylon147** [gugutab/TvOverlay#55]
I am trying to send the Notification with Image from HikVisions camera. The Camera Image URL is

 `http://Username:Password@192.168.1.64:80/ISAPI/Streaming/channels/1/picture?resolution=640x480
`
but somehow its not loading the image.

![WhatsApp Image 2024-12-25 at 3 04 45 PM](https://github.com/user-attachments/assets/a299416e-0d05-4ea4-b449-3049eaece9af)

From same camera I am able to Stream the Video successfully.

**Comment by cylon147** [comment]
@gugutab Kindly help me to fix this.

**Comment by sulsa** [comment]
@cylon147 - does the camera URL work if you open a network stream in VLC?

**Comment by cylon147** [comment]
@sulsa Yes, it works in the Browser. The URL is a snapshot image from a CCTV camera. The RTSP stream is working fine for the same Camera in the TVOverlay app.

![image](https://github.com/user-attachments/assets/a9d7e664-48f6-423e-8ce4-96fcb1edf683)

**Comment by nop0x900** [comment]
MJPEG stream not supported, app using exoplayer lib, and this lib d'nt work with MJPEG stream, using snapshot for hikvision cam.

**Comment by cylon147** [comment]
@nop0x900 Ohh! Thank you for the info. I will try different URLs or find a way to send RAW image files.
